src/main/java/randoop/main/GenTests.java

```diff
@@ -516,6 +522,75 @@ public boolean handle(String[] args) {
           codeWriter,
           GenInputsAbstract.regression_test_basename,
           "Regression");
+
+      HashSet<String> flakyTestNames = codeWriter.getFlakyTestNames();
+
+      if (flakyTestNames.size() > 0) {
+        List<ExecutableSequence> regressionSequences = explorer.getRegressionSequences();
+
+        // How many tests an operation occurs in.
+        // Tally occurrences of operations in all tests.
+        // Each method is counted exactly once if it appears in a sequence test.
+        Map<TypedOperation, Integer> testOccurrences =
+            tallyOperationsInSequences(regressionSequences);
+
+        // How many flaky tests an operation occurs in.
+        Map<TypedOperation, Integer> flakyOccurrences;
+
+        // TODO: cxing handle Error Test Sequence tallying.
+        // tallyOperationsInSequences(testOccurrences, explorer.getErrorTestSequences());
+
+        List<ExecutableSequence> flakySequences = new ArrayList<>();
+        for (String flakyTestName : flakyTestNames) {
+          int testNum = Integer.parseInt(flakyTestName.substring(4)); // length of "test"
+          // Tests start at 001
+          ExecutableSequence flakySequence = regressionSequences.get(testNum - 1);
+          flakySequences.add(flakySequence);
+        }
+
+        // Tally occurrences of operations in flaky tests.
+        // Each method is counted exactly once if it appears in a flaky test.
+        flakyOccurrences = tallyOperationsInSequences(flakySequences);
+
+        // tf-idf metric
+        // Our heuristic for probability of whether a method is flaky
```

Comment:
This is not a probability.  (It is a percentage, but it is used purely for ranking.)